torch/autograd/profiler.py

```diff
@@ -505,12 +543,27 @@ def _privateuse1_memory_usage(mem_record):
                             f_evt.device_index,
                             f_evt.time_range.end - f_evt.time_range.start,
                         )
+                    elif f_evt.device_type == DeviceType.XPU:
+                        fe.append_xpu_kernel(
+                            f_evt.name,
+                            f_evt.device_index,
+                            f_evt.time_range.end - f_evt.time_range.start,
+                        )
+                    elif f_evt.device_type == DeviceType.PrivateUse1:
+                        fe.append_privateuse1_kernel(
```

Comment:
Same as above, why do we need another `privateuse1_kernel` list?

Reply:
Commented as above. Thanks.